client/src/pages/LibrarySearchPage/LibrarySearchPage.js

```diff
@@ -34,33 +40,25 @@ class LibrarySearchPage extends Component {
     this.setState({ deleteModal: false });
   };
 
-  closeDeleteSuccessModal = () => {
-    this.setState({ deleteSuccessModal: false });
-  };
-
-  deleteEntity = (type, id) => {
-    const { libraryActions } = this.props;
-    libraryActions.remove(id, this.deleteSuccess, this.deleteFailure);
-  };
-
   deleteFailure = () => {
     window.scrollTo(0, 0);
+    console.log('error');
```

Comment:
Can you either remove this `console.log` message or make it a little more descriptive so it explains the error a little bit? Since this error will be appearing from a compiled React app once this is in production, the error message should be unique enough that you could find it by searching the files for that particular error message (in case the app is compiled in a way where the end user cannot see the individual files and line numbers where error messages were generated). Or, ideally, you should be able to tell what the message means if you just saw it in a list of console messages.

Reply:
I'll go ahead and remove it for now, since I added it in just to see if it was being triggered and just forgot to take it back out. However is there a particular way you would suggest throwing that error? Would you recommend just having a more detailed console message such as 'Error deleting file,' or is there a better way?

Reply:
@kmid5280 Thanks! I think removing it is just fine and I don't think anything is necessary here: There is an alert being shown telling the user the reason the delete failed (I believe that's done in the Redux action), and I don't think there's any extra information a developer would need to know in the JavaScript console.